### PR TITLE
pythonPackages.accuweather: init at 0.0.11

### PR DESCRIPTION
**Pull request by JamieMagee**
###### Motivation for this change

support `accuweather` in home assistant

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/accuweather/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    sha256 = "1sgbw9yldf81phwx6pbvqg9sp767whxymyj0ca9pwx1r6ipr080h";
+  };
+
+  buildInputs = [ pytestrunner ];
+  propagatedBuildInputs = [ aiohttp ];
+
+  checkInputs = [ pytest pytestcov pytest-asyncio ];
+  checkPhase = ''
+    ${python.interpreter} setup.py test
+  '';
```
Comment (jonringer) [inline pkgs/development/python-modules/accuweather/default.nix:21]: If you're going to use pytest, we have a helper for that
```suggestion
  nativeBuildInputs = [ pytestrunner ];
  propagatedBuildInputs = [ aiohttp ];

  checkInputs = [ pytestCheckHook pytestcov pytest-asyncio ];
```

**Comment by jonringer** [comment]
Please follow [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) and [manual#submitting-changes-making-patches ](https://nixos.org/nixpkgs/manual/#submitting-changes-making-patches) and squash the fix-up commits.

This can be done without `git rebase -i` by doing:
```
git reset HEAD~1                    # move fix-up commits into unstaged
git add -- pkgs/                    # move changes into staged
git commit --amend --no-edit        # add changes to previous commit
git push ... ... --force            # modify current PR branch
```

However, `git rebase -i` is a more powerful alternative, I created a small video demonstrating it's use [here](https://www.youtube.com/watch?v=OR8Q-4PLalc). A more indepth text tutorial can be found [here](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase)

**Comment by JamieMagee** [comment]
Thanks! Still getting familiar with contributing to nixpkgs.

I've made the required changes and set the minimum python version at 3.8. The project doesn't list a specific Python version, only Python 3. However, the tests fail on anything less than Python 3.8, and given the module is only 3 months old, it's not a surprise.

**Comment by JamieMagee** [comment]
@jonringer Ready for review

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 98644 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>python38Packages.accuweather</li>  </ul></details>

**Pull request merged**